### DISCUSSION TITLE
SKIN CHANGER LUCKPERMS

### DISCUSSION THREAD

**Discussion by hiyaraa** [edoren/SkinChanger#87] [Ideas]
Can only people with permissions use the skin set?

**Comment by edoren** [comment]
If you are asking about the base command, everyone in the server should be able to use the command.

Currently LuckPerms is not supported, but planned. I had an old PR for adding PermissionAPI support from Forge, but since migrating to use Architectury I have to find another way for Fabric and NeoForge

**Comment by CodeCatDeniz** [comment]
Currently on my server only operators can use the command, what can I do to fix this?
